webdriver/tests/bidi/script/evaluate/evaluate.py

```diff
@@ -24,3 +26,73 @@ async def test_interact_with_dom(bidi_session, top_context):
     assert result == {
         "type": "string",
         "value": "window.location.href: about:blank"}
+
+
+@pytest.mark.asyncio
+async def test_target_realm(bidi_session, top_context):
+    realms = await bidi_session.script.get_realms(context=top_context["context"])
+    default_realm = realms[0]["realm"]
```

Comment:
maybe add a fixture for the default_realm?

Reply:
Good idea! Added it